xbmc/guilib/GraphicContext.cpp

```diff
@@ -983,16 +984,17 @@ void CGraphicContext::SetMediaDir(const std::string &strMediaDir)
   m_strMediaDir = strMediaDir;
 }
 
+
+void CGraphicContext::SetStereoMode(RENDER_STEREO_MODE mode)
+{
+  m_stereoMode = mode;
+  g_application.m_pPlayer->TriggerUpdateResolution();
```

Comment:
Yes, you can call those but must ensure that those are called from the main thread. Don't use TriggerUpdateResolution. I need to rework because there is already a dependency to player in gfx context. I did not take care when I changed g_application.m_pPlayer->GetRenderResolution to TriggerUpdateResoltution. I should have eliminated it that time.